e2e-tests/cypress/tests/integration/channels/account_settings/profile/email_spec.ts

```diff
@@ -224,7 +224,7 @@ describe('Profile > Profile Settings > Email', () => {
             // # Do login
             cy.get('#input_loginId').should('be.visible').clear().type(email);
             cy.get('#input_password-input').should('be.visible').type(testUser.password);
-            cy.get('#saveSetting').should('not.be.disabled').click();
+            cy.findByTestId('saveSetting').should('not.be.disabled').click();
```

Comment:
We have several test based on text content... that failed and got skipped because of text changes :sweat_smile: That being said, I agree with you, we should try to go more in that direction.